### PR TITLE
Added printf command

### LINKED ISSUE #63: [Add] shellutils/printf command
**If you want to develop the command described in this Issue title, please comment on this issue**

### PR DESCRIPTION
**Pull request by k-avy**
## Description 
- Added printf command.
- fixes #63 

## Working

image

### CODE REVIEW

File: internal/applets/shellutils/printf/printf.go [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+)
+
+const cmdName string = "printf"
+const version = "1.0.2"
+
+var osExit = os.Exit
+
+type options struct {
+	Version bool `short:"v" long:"version" description:"Show Printf command version"`
+}
```
Comment (nao1215) [inline internal/applets/shellutils/printf/printf.go:18]:
```suggestion
const cmdName string = "printf"
const version = "1.0.0"

var osExit = os.Exit

type options struct {
	Version bool `short:"v" long:"version" description:"Show printf command version"`
}
```

**Review by nao1215 — CHANGES_REQUESTED** [review]
You have made some great additions.
I would like to ask you to correct a few things. I am giving you a suggestion.

There is a missing spec file for testing, but I assume that this is not a problem here.

**Comment by nao1215** [comment]
If you intend to add test code, please work through the steps below.
1.  Add test/it/spec/printf_spec.sh
2. Add test code at test/it/spec/printf_spec.sh
3. Run integration test.
```
$ cd <project root>
$ make it

(in docker)
$ $ ./do_integration_test.sh 
```

**Comment by nao1215** [comment]
@k-avy 
After the GitHub Actions error (Build, Integration Test) is fixed, I will merge this PR.

**Comment by k-avy** [comment]
Thank you for reviewing i will fix the error and push it again.

**Comment by k-avy** [comment]
I have fixed the error and removed the test for now as i dont know how to do that.

**Comment by nao1215** [comment]
@k-avy 
You have done a very good job.
I will merge your pr.

**Comment by k-avy** [comment]
@nao1215 
Thank you for the appreciation.

**Pull request merged**